docs/content/posts/2024-10-30-release-kubernetes-operator-1.10.0.md

```diff
@@ -0,0 +1,78 @@
+---
+title:  "Apache Flink Kubernetes Operator 1.10.0 Release Announcement"
+date: "2024-10-25T18:00:00.000Z"
+authors:
+- mateczagany:
+  name: "Mate Czagany"
+- 1996fanrui:
+  name: "Rui Fan"
+  twitter: "1996fanrui"
+aliases:
+- /news/2024/10/30/release-kubernetes-operator-1.10.0.html
+---
+
+The Apache Flink community is excited to announce the release of Flink Kubernetes Operator 1.10.0!
+
+The release includes several improvements to the autoscaler, and introduces a new Kubernetes custom resource called FlinkStateSnapshot to manage job snapshots.
+The process of job upgrades has also been enhanced which makes it possible to now use the last-state upgrade mode with session jobs.
+
+We encourage you to [download the release](https://flink.apache.org/downloads.html) and share your experience with the
+community through the Flink [mailing lists](https://flink.apache.org/community.html#mailing-lists) or
+[JIRA](https://issues.apache.org/jira/browse/flink)! We're looking forward to your feedback!
+
+## Highlights
+
+### FlinkStateSnapshot
+
+With this version comes also a new custom resource called FlinkStateSnapshot. 
+This is used to describe savepoint or checkpoint for a Flink job. 
+The savepoint/checkpoint fields found in FlinkDeployment and FlinkSessionJob status are therefore deprecated, and the operator will create new FlinkStateSnapshot resources for periodic, update and manual savepoints/checkpoints.
+
+Users can also create new FlinkStateSnapshot resources, which will instruct the operator to trigger new checkpoint/savepoint.
+
+This new feature is enabled by default, unless disabled by setting `kubernetes.operator.snapshot.resource.enabled` to false or if the FlinkStateSnapshot CRD was not found on the Kubernetes cluster.
+
+You can read more about this feature [here](https://nightlies.apache.org/flink/flink-kubernetes-operator-docs-release-1.10/docs/custom-resource/snapshots/).
+
+
+### Last-State Upgrade Mode
+
+For deployments using last-state upgrade mode, the operator - instead of forcefully deleting the JobManager pods to trigger a restart - will cancel the job via REST API and extract the last checkpoint info after cancellation if the job is healthy.
+This change makes it possible to finally use the last-state upgrade mode for session jobs as well.
```

Comment:
The default behaviour for applications is still to simply delete the JM to trigger the failover mechanism. Overall when this is possible to do , it's the simplest, fastest and more robust. 

But when HA is off or HA metadata does not exist, cancelling as a mechanism will be used (+ for session jobs as you wrote)

Reply:
Changed it and rebuilt website, please let me know if there are any other inaccuracies